dart/welcome_email/README.md

```diff
@@ -0,0 +1,37 @@
+# 📧 Sending Welcome Emails using Mailgun's Email API
+A sample Dart Cloud Function for sending a welcome email to a newly registered user.
+
+## 📝 Environment Variables
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+
+* **MAILGUN_API_KEY** - API Key for Mailgun 
+* **MAILGUN_DOMAIN** - Domain Name from Mailgun
+
+## 🚀 Building and Packaging
+
+To package this example as a cloud function, follow these steps.
+
+* Ensure that your folder structure looks like this 
+```
+.
+├── main.dart
+├── .appwrite
+└── pubspec.yaml
+```
+
```

Comment:
We should also include a section in the readme that talks about which versions of the dart runtime, this example can run on. 
Since we're having only 1 folder for all the different dart versions, we should have some sort of a note in the readme that says

> This example is compatible with dart-2.15 , dart-2.13 

Something like this, but probably more cleaner